tests/rkt_caps_test.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2015 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"os"
+	"testing"
+
+	"github.com/coreos/rkt/Godeps/_workspace/src/github.com/ThomasRooney/gexpect"
+)
+
+var capsTests = []struct {
+	rktCmd string
+	expect string
+}{
+	{
+		`../bin/rkt --debug --insecure-skip-verify run ./rkt-inspect-print-caps.aci`,
+		"Effective: 0xfdecafff",
```

Comment:
Also, this test case fails. Why? Is it that semaphore doesn't provide all caps rkt needs?